### PR TITLE
Improve abuse report wording

### PR DESCRIPTION
**Pull request by rumpelsepp**
@Razer6

### CODE REVIEW

File: app/views/abuse_reports/new.html.haml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 - page_title "Report abuse"
 %h3.page-title Report abuse
-%p Please use this form to report users who create spam issues or comments or who otherwise behave inappropriately.
+%p Please use this form to report users who create spam issues, comments or behave inappropriately.
```
Comment (stanhu) [inline app/views/abuse_reports/new.html.haml]: Another comma needed between `comments` and `or`.

Comment (rumpelsepp) [inline app/views/abuse_reports/new.html.haml]: Just for me (as a non native english speaker): The comma between `comments` and `or` is optional, isn't it?

**Comment by rumpelsepp** [comment]
:+1:

File: app/views/abuse_reports/new.html.haml (hunk 1 above)
Comment (stanhu) [inline app/views/abuse_reports/new.html.haml]: Sorry, I misread the sentence. This was correct.

Comment (rumpelsepp) [inline app/views/abuse_reports/new.html.haml]: :tada: :blush:

**Pull request merged**